react/src/components/DesktopAppDownloadModal.tsx

```diff
@@ -0,0 +1,64 @@
+import { useSuspendedBackendaiClient, useAppDownloadMap } from '../hooks';
+import BAIModal, { BAIModalProps } from './BAIModal';
+import Flex from './Flex';
+import { Select, Button, Descriptions, Tooltip } from 'antd';
+import { map, toUpper } from 'lodash';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface DesktopAppDownloadModalProps extends BAIModalProps {
+  onRequestClose?: () => void;
+}
+
+const DesktopAppDownloadModal: React.FC<DesktopAppDownloadModalProps> = ({
+  onRequestClose,
+  ...baiModalProps
+}) => {
+  const { t } = useTranslation();
+  const baiClient = useSuspendedBackendaiClient();
```

Comment:
18 - 19 line also could be moved into useAppDownloadMap I guess. appDownloadUrl is referenced only for the hook

Reply:
Oops! I missed. It is fixed.